src/main/java/io/vertx/proton/ProtonClientOptions.java

```diff
@@ -40,6 +40,7 @@
   private Set<String> enabledSaslMechanisms = new LinkedHashSet<>();
 
   private int heartbeat;
+  private int maxFrameSize;
```

Comment:
The equals and hashCode methods needs to be updated to account for the new field.